Core/CMP API Specification.md

```diff
@@ -482,94 +480,22 @@ The `getField` command can be used to receive a specific field out of a certain
   </tr>
   <tr>
     <td><code>callback</code></td>
-    <td>not used</td>
-    <td></td>
+    <td>function</td>
+    <td>function (data: mixed or null, success)</td>
   </tr>
   <tr>
     <td><code>parameter</code></td>
     <td>string</td>
     <td>API Prefix string + "." (dot) + fieldname</td>
   </tr>
-  <tr>
-    <td><code>return</code></td>
-    <td>mixed or null</td>
-    <td>Section string parsed into an object according to the API specification</td>
-     </td>
-     </td>
-  </tr>
  </table>
 
 
 
 For example, a client can ask the CMP to get the last updated field from the IAB TCF v2.0 TCData. 
 
 ```javascript
-var s = __gpp('getField',null, "tcfeuv2.LastUpdated");
-```
-
-______
-#### `getGPPData` <a name="getgppdata"></a>
```

Comment:
doesn't __gpp('getSection',null, "tcfeuv2"); raise the same concerns as getTCData and need deprecation as well?